assets/src/edit-story/components/library/panes/media/mediaEditDialog.js

```diff
@@ -184,6 +196,15 @@ function MediaEditDialog({ resource, onClose }) {
           </Video>
         )}
         <MetadataTextContainer>
+          {parsedDate.isValid() && (
+            <MediaDateText>
+              {sprintf(
+                /* translators: %s: upload date of media item. */
+                __('Uploaded: %s', 'web-stories'),
+                parsedDate.format('MMMM DD, YYYY')
```

Comment:
This should use the date format from the WordPress settings.

I'll probably file an issue for this to tackle separately, as it requires some additional work.

Reply:
Actually, you should be able to use `parsedDate.format('LL')` or `parsedDate.format('LLL')`. Those formats come from the WordPress settings.

Reply:
Opened #2334 for context.

In the meantime:

```suggestion
                parsedDate.format('LL')
```

Reply:
I tried `.format('LL')` but that gave me a weird unformatted string instead.

![Screen Shot 2020-06-11 at 2 28 13 PM](https://user-images.githubusercontent.com/15134432/84345131-dd816680-abef-11ea-84b6-c2027e834442.png)

Is it because it's a local instance (`npm run dev`) and the locales aren't recognised?

Reply:
Odd 🤔 I'll take a look

Reply:
Left it as ('MMMM DD, YYYY') for now, will be fixed in #2334.